From 5d2db4358c6de8c99c4d667499df029d293a8e38 Mon Sep 17 00:00:00 2001
From: Thomas Meyer <tmeyer@student.42.fr>
Date: Sun, 20 Nov 2022 11:09:47 +0100
Subject: [PATCH 1/1] Updating version from 6.1-rc3 to 6.1-rc3-thor_kernel

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 28026d1ebb9d..91efb81e9047 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 1
 SUBLEVEL = 0
-EXTRAVERSION = -rc3
+EXTRAVERSION = -rc3-thor_kernel
 NAME = Hurr durr I'ma ninja sloth
 
 # *DOCUMENTATION*
-- 
2.38.1

